CardScan/CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -402,7 +408,13 @@ public protocol TestingImageDataSource: AnyObject {
         if (self.sendScanStats) {
             // fire and forget
             let scanStatsPayload = self.ocrMainLoop()?.scanStats.createPayload() ?? ScanStats().createPayload()
-            ScanApi.uploadScanStats(payload: scanStatsPayload, completion: { _, _ in })
+            ScanApi.uploadScanStats(payload: scanStatsPayload, completion: { response, error in
+                guard let status = response?.status, status == "ok" else {
```

Comment:
is this guaranteed to always be lowercase?

Reply:
yea it is https://github.com/getbouncer/cardscan-ios/blob/f37f24b041906560242eeb466911e29f5d8da20f/CardScan/CardScan/Classes/API.swift#L77-L78